pyriemann/clustering.py

```diff
@@ -2,7 +2,25 @@
 import numpy
 from sklearn.base import (BaseEstimator, ClassifierMixin, TransformerMixin,
                           ClusterMixin)
-from sklearn.cluster._kmeans import _init_centroids
+
+try:
+    from sklearn.cluster._kmeans import _init_centroids
+except ImportError:
+    # Workaround for scikit-learn v0.24.0rc1+
+    # See issue: https://github.com/alexandrebarachant/pyRiemann/issues/92
+    from sklearn.cluster import KMeans
+
+    def _init_centroids(X, n_clusters, init, random_state, x_squared_norms):
+        if random_state is not None:
```

Comment:
Uh, I'm a bit late with noticing this, but this seems wrong?

Should probably be `if random_state is None:`, right?